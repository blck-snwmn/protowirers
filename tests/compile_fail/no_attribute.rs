use protowirers::*;

#[derive(Proto)]
struct Sample {
    s: u32,
}

fn main() {}
