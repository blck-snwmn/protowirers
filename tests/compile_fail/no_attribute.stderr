error: #[def(...)] attribute is required
 --> $DIR/no_attribute.rs:5:5
  |
5 |     s: u32,
  |     ^
